pom.xml

```diff
@@ -393,8 +393,8 @@
         <version.org.hibernate.validator>6.0.22.Final</version.org.hibernate.validator>
         <version.org.hornetq>2.4.7.Final</version.org.hornetq>
         <!-- n.b. Infinispan Server version used for integration testing is defined separately in testsuite/integration/clustering/pom.xml -->
-        <version.org.infinispan>11.0.10.Final</version.org.infinispan>
-        <version.org.infinispan.protostream>4.3.5.Final</version.org.infinispan.protostream>
+        <version.org.infinispan>12.1.1.Final</version.org.infinispan>
+        <version.org.infinispan.protostream>4.4.0.Beta3</version.org.infinispan.protostream>
```

Comment:
4.4.0.Final is released

Reply:
Looks like that commit got lost in a rebase and force push.